$MirOS: ports/net/djbdns/patches/patch-man_tinydns-data_8,v 1.3 2006/01/15 12:57:46 tg Exp $
--- man/tinydns-data.8.orig	Thu Oct 23 07:47:10 2003
+++ man/tinydns-data.8	Wed Jul 26 23:42:43 2006
@@ -1,4 +1,4 @@
-.TH tinydns-data 8
+.TH tinydns-data 8 "June 13, 2003" "" "DJBDNS Operation Guide"
 
 .SH NAME
 tinydns-data \- data tool for tinydns
@@ -53,7 +53,7 @@ Blank lines are ignored.
 
 Each line contains a
 .I ttl
-(``time to live'')
+("time to live")
 specifying the number of seconds that the line's DNS records may be cached.
 Beware that cache times below 300 seconds
 will be treated as 300 by some clients,
@@ -74,7 +74,7 @@ the line will be ignored before that tim
 If 
 .I ttl
 is zero,
-the timestamp is an ending time (``time to die'')
+the timestamp is an ending time ("time to die")
 for the information in the line;
 .BR tinydns (8)
 dynamically adjusts 
@@ -144,14 +144,14 @@ Name server for our domain 
 .B tinydns-data
 creates
 .IP
-an NS (``name server'') record
+an NS ("name server") record
 showing 
 .IR x .ns.\fIfqdn\fR
 as a name server for
 .IR fqdn ;
 .P
 .IP
-an A (``address'') record showing
+an A ("address") record showing
 .I ip
 as the IP address
 of
@@ -159,7 +159,7 @@ of
 and
 .P
 .IP
-an SOA (``start of authority'') record for 
+an SOA ("start of authority") record for 
 .I fqdn
 listing 
 .IR x\fR.ns.\fIfqdn
@@ -304,33 +304,50 @@ and an A record showing
 as the IP address of
 .IR ns7.panic.mil .
 
-.RI = fqdn\fR:\fIip\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
+.RI = fqdn\fR:\fIip4\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
+.br
+.RI 6 fqdn\fR:\fIip6\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
 
 Host
 .I fqdn
-with IP address 
-.IR ip .
+with IPv4/IPv6 address 
+.I ip4
+or
+.IR ip6 .
 
 .B tinydns-data
 creates
 .IP
-an A record showing 
-.I ip
+an A (or AAAA) record showing 
+.I ip4
+(or
+.IR ip6 )
 as
 the IP address of 
 .IR fqdn
 and
 .P
 .IP
-a PTR (``pointer'') record showing 
+a PTR ("pointer") record showing 
 .I fqdn
 as
 the name of 
 .IR d.c.b.a .in-addr.arpa
 if 
-.I ip
+.I ip4
 is 
 .IR a.b.c.d .
+
+A PTR record of
+.IR 0.1.2.3.4.5.6.7.8.9.a.b.c.d.e.f.f.e.d.c.b.a.9.8.7.6.5.4.0.1.2.3 .ip6.arpa
+and (deprecated fallback)
+.IR 0.1.2.3.4.5.6.7.8.9.a.b.c.d.e.f.f.e.d.c.b.a.9.8.7.6.5.4.0.1.2.3 .ip6.int
+linking to
+.I fqdn
+is created if
+.I ip6
+is
+.IR 3210:4567:89AB:CDEF:FEDC:BA98:7654:3210 .
 .P
 
 Remember to specify name servers for some suffix of
@@ -358,7 +375,9 @@ and a PTR record showing
 as the name of
 .IR 108.7.8.1.in-addr.arpa .
 
-.RI + fqdn\fR:\fIip\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
+.RI + fqdn\fR:\fIip4\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
+.br
+.RI 3 fqdn\fR:\fIip6\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
 
 Alias
 .I fqdn
@@ -366,6 +385,8 @@ with IP address 
 .IR ip .
 This is just like 
 .RI = fqdn\fR:\fIip\fR:\fIttl\fR
+and (IPv6 case)
+.RI 6 fqdn\fR:\fIip\fR:\fIttl\fR
 except that 
 .B tinydns-data
 does not create the PTR record.
@@ -376,8 +397,12 @@ returns addresses
 (from
 .I +
 or
+.I 6
+or
 .I =
 or
+.I 3
+or
 .I @
 or
 .I .
@@ -405,7 +430,7 @@ Mail exchanger for 
 .B tinydns-data
 creates
 .IP
-an MX (``mail exchanger'') record
+an MX ("mail exchanger") record
 showing 
 .IR x\fR.mx.\fIfqdn
 as a mail exchanger for 
@@ -471,7 +496,7 @@ The line is ignored.
 
 .RI ' fqdn\fR:\fIs\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
 
-TXT (``text'') record for 
+TXT ("text") record for 
 .IR fqdn .
 .B tinydns-data
 creates a TXT record for 
@@ -499,7 +524,7 @@ pointing to the domain name 
 
 .RI C fqdn\fR:\fIp\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
 
-CNAME (``canonical name'') record for 
+CNAME ("canonical name") record for 
 .IR fqdn .
 .B tinydns-data
 creates a CNAME record for 
@@ -518,8 +543,8 @@ use
 .RI + fqdn
 instead.
 Remember the wise words of Inigo Montoya:
-``You keep using CNAME records.
-I do not think they mean what you think they mean.''
+"You keep using CNAME records.
+I do not think they mean what you think they mean."
 
 .RI Z fqdn\fR:\fImname\fR:\fIrname\fR:\fIser\fR:\fIref\fR:\fIret\fR:\fIexp\fR:\fImin\fR:\fIttl\fR:\fItimestamp\fR:\fIlo\fR
 
@@ -704,3 +729,11 @@ if the database were rebuilt periodicall
 tinydns(8)
 
 http://cr.yp.to/djbdns.html
+.br
+http://www.fefe.de/dns/
+.br
+http://lifewithdjbdns.org/
+.br
+http://tinydns.org/
+.br
+http://www.guinix.com/technote/dualdns.html
